plugins/node/opentelemetry-instrumentation-ioredis/src/utils.ts

```diff
@@ -121,11 +122,11 @@ export const traceSendCommand = (
       const origResolve = cmd.resolve;
       /* eslint-disable @typescript-eslint/no-explicit-any */
       cmd.resolve = function (result: any) {
-        try {
-          config?.responseHook?.(span, cmd.name, cmd.args, result);
-        } catch {
-          // we have nothing to do with exception from hook
-        }
+        safeExecuteInTheMiddle(
+          () => config?.responseHook?.(span, cmd.name, cmd.args, result),
+          () => {},
```

Comment:
Sorry i didn't catch this before but i think it could be better if we could log an error so the user know that its hook is failing

Reply:
makes sense. added logging on error :)